pkgs/development/python-modules/searx.patch

```diff
@@ -16,7 +16,8 @@ index 0d2f61b..46481b3 100644
  pyasn1-modules==0.0.8
```

Comment:
I don’t want to lose semantic versioning. If a major version changes normally the API isn’t interoperable any more and since this is python, we don’t know until it crashes sometime at runtime. Even more, since most users have a lot of search engine plugins turned off by default, it might crash anytime for anyone.